### DISCUSSION TITLE
How do you handle bounce messages?

### DISCUSSION THREAD

**Discussion by Pacerino** [postalserver/postal#2743] [Help with using Postal]
We now use Postal in a productive environment and are reasonably satisfied with it. Now I just had the question of how we can process bounces and I wanted to turn to the community with this question. So my question to everyone is, how do you process your bounces? I've already discovered Abuse.io, but it doesn't look well maintained.

I am mainly concerned with the automatic messages from the e-mail provider when, for example, the mailbox is full.

Thank you for your answers!

**Comment by willpower232** [comment]
Usually just implement webhooks so whatever is sending the messages can handle things as necessary, don't really need to go much further than that.
